### ISSUE TITLE
Support assertionResults

### ISSUE THREAD

**Issue by dkelosky** [dkelosky/jest-stare#49]
#45 noted that --json creates a different json object to parse.  This issue is to track support for that format.

**Comment by dkelosky** [comment]
This seems like a jest bug. 

They have an object like this:
```
    "testResults": [
        {
            "assertionResults": [
                {
                    "ancestorTitles": [
                        "Example test suite for data driven test"
                    ],
```
Instead of this:
```
    "success": false,
    "testResults": [
        {
            "testResults": [
                {
                    "ancestorTitles": [
                        "Example test suite for data driven test"
                    ],
```

The inner `testResults` array is of type `AssertionResults[]` - however, jest is providing the object key of `assertionResults`.  I'll add a simple hack to attempt to read `assertionResults` if `testResults` are missing and treat them as `assertionResults`.  Warnings will be issued in the browser console.

**Comment by dkelosky** [comment]
"fixed" via #55
